extensions/csrf-reactive/runtime/src/main/java/io/quarkus/csrf/reactive/runtime/CsrfHandler.java

```diff
@@ -173,7 +166,6 @@ private static Response badClientRequest() {
     @ServerResponseFilter
```

Comment:
You can't have `@ServerRequestFilter` and `@ServerResponseFilter` in a `ServerHandler`

Reply:
It seems to work but sure I can move it out to its own standalone class

Reply:
Well, sorry, I don't think I have the tests going there yet, so I'll just move it